stan/math/rev/scal/fun/is_nan.hpp

```diff
@@ -3,6 +3,7 @@
 
 #include <stan/math/rev/meta.hpp>
 #include <stan/math/rev/core.hpp>
+#include <stan/math/prim/meta.hpp>
```

Comment:
This shouldn't be necessary -- it's pulled in through `stan/math/rev/meta.hpp`. If it doesn't build with this line removed, there's a problem with our includes that we need to fix.